### PR TITLE
Adds a jester suit and a white dress, both by Nienhaus.

### PR DESCRIPTION
**Pull request by theOperand**
Inhands not included.

niensprites

### CODE REVIEW

**Comment by Nienhaus** [comment]
♥

**Comment by Miauw62** [comment]
Alright, pics added and this works now.

**Comment by Nienhaus** [comment]
Oh man, it'd cool to add shoes that are actually invisible on-mob for things like dresses.

**Comment by Miauw62** [comment]
I think white shoes would work too.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -469,4 +469,21 @@
...
\ No newline at end of file
+	item_color = "plaid_purple"
+
+/obj/item/clothing/under/whitedress
+	name = "white dress"
+	desc = "A fancy white dress."
+	icon_state = "white_dress"
+	item_state = "w_suit"
+	item_color = "white_dress"
+	fitted = 0
```
Comment (Incoming5643) [inline code/modules/clothing/under/miscellaneous.dm]: these use defines now: You want to change the 0 here to NO_FEMALE_UNIFORM. You might want to see if it looks ok with FEMALE_UNIFORM_TOP first though.

**Comment by Nienhaus** [comment]
That'd still block the dress

**Comment by RandomMarine** [comment]
Maybe make the dress clothing/suit instead.

**Comment by Nienhaus** [comment]
But the top part is translucent so the jumpsuit would be shown under.

**Comment by Incoming5643** [comment]
HIDEUNIFORM

**Comment by RemieRichards** [comment]
Cute :+1:

**Comment by Atskadan** [comment]
not lifeweb jester uniform :-1:

**Comment by Nienhaus** [comment]
I told Miauw to give me one to sprite.

**Comment by Miauw62** [comment]
i will not let anybody pay tribute to "muh hardcoer no casuel plz" lifeweb on this repo, unless possibly if it makes the author mad.

any more opinions on whether the dress should be suit or uniform?

**Comment by Nienhaus** [comment]
You could try it if you can somehow get the uniform to be hidden.

**Comment by Atskadan** [comment]
lmfao miauw confirmed for casual baddie
:salt:

**Comment by RemieRichards** [comment]
Merge conflict

**Comment by Miauw62** [comment]
Conflicts resolved, commits squashed.
Ready 4 merge @WJohn (the code is fine)

**Comment by WJohn** [comment]
:+1:

**Comment by Nienhaus** [comment]
Hey @Miauw62 "We can’t automatically merge this pull request."

**Comment by WJohn** [comment]
Hm, travis isn't even trying to take another shot at it for automerging.

**Comment by Miauw62** [comment]
>conflicts again

kill me

**Comment by Miauw62** [comment]
alright, there we go

**Comment by Miauw62** [comment]
apparently the squash went wonky but idc

**Comment by Miauw62** [comment]
@WJohn

**Comment by Miauw62** [comment]
<3

**Comment by Nienhaus** [comment]
:D

**Comment by Atskadan** [comment]
lol miauw is so fucking mad

**Pull request merged**